File: Python/1 Operações/bala.py
import math
x1 = float(input())
y1 = float(input())

x2 = float(input())
y2 = float(input())

ans = math.sqrt( math.pow(x2-x1, 2) + math.pow(y2- y1, 2))
print(f"{ans:.2f}")

File: Fup is Fun/C/1 Operações/bala.c
#include <stdio.h>
#include <math.h>

int main() {
    float x1,y1,x2,y2;
    scanf("%f %f", &x1,&y1);
    scanf("%f %f", &x2,&y2);

    float ans = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
    printf("%.2f\n",ans);
}
